### ISSUE TITLE
Reproducibility

### ISSUE THREAD

**Issue by zwx8981** [cientgu/GIQA#11]
Hi, I have tried to reproduce the results on FFHQ, where I only achieved an accuracy of about 68% using GMM-GIQA.  I have tried your pretrained GMM model and I can get the result reported in your paper, this verifies that there is no problem in the feature extraction part. Then I switched to use KNN-GIQA, and I get a poor result of roughly 58%, this may manifests that the there is difference of training data. I directly use the 70000 cropped and aligned 1024X1024 images of FFHQ, is there any inconsistency with your practice?

**Comment by cientgu** [comment]
I do not crop and align them, only directly resizing them to 256*256. Sorry that I forget the released model uses the 70000 images or 140000 images(with flip augmentation). But I'm sure using 140000 images is slightly better than 70000 images.

**Comment by zwx8981** [comment]
Thanks. You mean you use the original images from Flicker  (in-the-wild-images with a size of 955GB) as indicated in https://github.com/NVlabs/ffhq-dataset ?

**Comment by cientgu** [comment]
No, I use the images1024x1024 (89.1G), but downsample them to 256*256.  I misunderstood your meaning of crop and align.

**Comment by zwx8981** [comment]
I got it, many thx.
